addons/web/static/src/legacy/js/core/smooth_scroll_on_drag.js

```diff
@@ -129,9 +131,19 @@ const SmoothScrollOnDrag = Class.extend(mixins.ParentedMixin, {
         this.options.jQueryDraggableOptions.scroll = false;
         this.options.disableHorizontalScroll = this.options.disableHorizontalScroll || false;
         const draggableOptions = Object.assign({}, this.options.jQueryDraggableOptions, {
-            start: (ev, ui) => this._onSmoothDragStart(ev, ui, this.options.jQueryDraggableOptions.start),
+            start: (ev, ui) => {
+                this._onSmoothDragStart(ev, ui, this.options.jQueryDraggableOptions.start);
+                if (insideIframe) {
+                    window.addEventListener('mouseup', this._onParentWindowMouseup.bind(this), {once: true});
```

Comment:
You bind to this for both `addEventListener` and `removeEventListener`, so the remove cannot work :grimacing:

Reply:
> You bind to this for both `addEventListener` and `removeEventListener`, so the remove cannot work grimacing

Indeed :neutral_face: I did not test in the right way
I made the change